nncf/common/graph/transformations/commands.py

```diff
@@ -103,25 +105,25 @@ class TargetPoint:
     will be applied.
     """
 
-    def __init__(self, target_type: TargetType):
+    def __init__(self, target_type: TargetType, target_node_name: NNCFNodeName):
         """
         Constructor.
 
         :param target_type: Type of the target point.
         """
         self._target_type = target_type
+        self.target_node_name = target_node_name
```

Comment:
Could you please add a docstring for the `target_node_name` parameter?

Reply:
Done